bridge/src/lib.rs

```diff
@@ -212,6 +241,123 @@ methods!(
 
         NilClass::new()
     }
+
+    fn start_test_server(
```

Comment:
For both these functions, I'd re-use the config objects and pass them in rather than have the mega param list.

Clippy (I thought we had that turned on in here?) would whine about so many parameters, for good reason. 

You might also consider more closely mirroring the config structure from core since some of these parts are re-used (at least, in the Rust layer, I get not doing that at the Ruby level).

Reply:
I'd love to, but unfortunately `rutie` doesn't have bindings for Ruby's `Struct` or something similar that would allow me to pass a single object rather than individual params. I can use an `Array` or `Hash`, but that's arguably worse, because we lose typing and need to cast everything instead.

Clippy is on, we run `cargo clippy --workspace --all-features --all-targets -- -D warnings` in CI, but that hasn't complained.

I can potentially get rid of the `TestServerConfig` in favour of `ephemeral_server::TestServerConfigBuilder`, but that'll mean I'll need to compose it in `lib.rs` rather than further down, losing that separation between the part that just deals with Ruby and the part than deals with Core without fusing the two. Which I'm not sure is better, but please let me know if you see benefits of this. And also `ephemeral_server::EphemeralExe` is a bit of a pain to work with.